easybuild/easyconfigs/f/FHI-aims/FHI-aims-200112_2-intel-2019b.eb

```diff
@@ -0,0 +1,49 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Authors::   Dugan Witherick (University of Warwick)
+#
+##
+easyblock = 'CMakeMake'
+
+name = 'FHI-aims'
+version = '200112_2'
+
+homepage = 'https://aimsclub.fhi-berlin.mpg.de/'
+description = """FHI-aims is an efficient, accurate all-electron,
+full-potential electronic structure code package for computational molecular
+and materials science (non-periodic and periodic systems). The code supports
+DFT (semilocal and hybrid) and many-body perturbation theory. FHI-aims is
+particularly efficient for molecular systems and nanostructures, while
+maintaining high numerical accuracy for all production tasks. Production
+calculations handle up to several thousand atoms and can efficiently use (ten)
+thousands of cores. 
+"""
+
+toolchain = {'name': 'intel', 'version': '2019b'}
+toolchainopts = {'opt': True, 'precise': True}
+
+# The source code must be downloaded manually from the FHI-aims club.
+sources = ['%(namelower)s.%(version)s.tgz']
+checksums = ['b591b09c7cb2e0befe59067ae59a63f522a9b5200a4073018f616aea657704d6']
+
+builddependencies = [('CMake', '3.15.3')]
+
+configopts = ' -DCMAKE_Fortran_COMPILER="$MPIF90" '
+configopts += ' -DLIBS="mkl_scalapack_lp64 mkl_blacs_intelmpi_lp64 mkl_intel_lp64 mkl_sequential mkl_core" '
+configopts += ' -DLIB_PATHS="$CMAKE_LIBRARY_PATH" '
+configopts += ' -DCMAKE_C_FLAGS="$CFLAGS -ip" '
+configopts += ' -DCMAKE_Fortran_FLAGS="$FFLAGS -ip" '
+configopts += ' -DFortran_MIN_FLAGS="-O0 -fp-model precise" '
+configopts += ' -DTARGET_NAME="aims.x" '
+
+postinstallcmds = ["cp -ar %(builddir)s/%(namelower)s.%(version)s/{CHANGELOG.md,doc,external} %(installdir)s/",
+                   "cp -ar %(builddir)s/%(namelower)s.%(version)s/{regression_tests,species_defaults} %(installdir)s/",
+                   "cp -ar %(builddir)s/%(namelower)s.%(version)s/{testcases,utilities} %(installdir)s/"]
+
+sanity_check_paths = {
+    'files': ['bin/aims.x'],
+    'dirs': [],
+}
+
+moduleclass = 'math'
```

Comment:
nitpicking: DFT is usually classified as `chem`, no?

Reply:
Sorry, was away. Yes I think so, although I've got engineers using it so didn't want to insult anyone :-) I will change it to chem and resubmit.